nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/test/java/org/apache/nifi/processors/mongodb/GetMongoIT.java

```diff
@@ -61,9 +64,9 @@
     static {
         CAL = Calendar.getInstance();
         DOCUMENTS = Lists.newArrayList(
-            new Document("_id", "doc_1").append("a", 1).append("b", 2).append("c", 3),
-            new Document("_id", "doc_2").append("a", 1).append("b", 2).append("c", 4).append("date_field", CAL.getTime()),
-            new Document("_id", "doc_3").append("a", 1).append("b", 3)
+                new Document("_id", "doc_1").append("a", 1).append("b", 2).append("c", 3),
```

Comment:
Was this done by IDE auto-indentation or because the earlier version did not match the general indentation? There are also similar places in the other classes.

Reply:
Yes, it is due to earlier version mismatch, for continuous indentation it is 8 spaces in some places  and 4 at some. However, I have indented as before.